### DISCUSSION TITLE
[mac v3.0.1] Issue: oversize rendered latex in edit mode

### DISCUSSION THREAD

**Discussion by yixin0829** [tamlok/vnote#1784] [Q&A]
tried v3.2.0 and v3.1.0 as well. Any workaround to avoid this?
![image](https://user-images.githubusercontent.com/56566212/120929198-1f83a200-c6b6-11eb-8a26-79c3fd0f9d45.png)

**Comment by tamlok** [comment]
Check if the scale factor of your read mode is larger than `1.0`. If so, restore it to the default `1.0` and check again. If you feel the text size of the read mode by default is too small, go to the theme folder and modify the `web.css` style sheet file to have a bigger font.

**Reply by yixin0829** [reply]
sry i was talking about the edit mode. see the screenshot below:
![image](https://user-images.githubusercontent.com/56566212/121614643-264f4380-ca2d-11eb-80a5-c0ddfdb4b8a7.png)
the rendered latex feels disproportional. in the read mode it's fine.

**Reply by tamlok** [reply]
Please check the scale factor of the **READ** mode. The in-place preview in the edit mode is just a snapshot of the read mode.

Just do as what I said in the first reply.

**Reply by yixin0829** [reply]
it was 1.00 but the problem was fixed when I tried to reset it again. Thank you!
